main.go

```diff
@@ -27,9 +28,18 @@ var (
 	helpFlag         = flag.Bool("help", false, "")
 
 	grafana = NewGrafanaUpdater(*grafanaUrl, *grafanaUsername, *grafanaPassword)
+
+	configMapUpdates = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "grafana_watcher_configmap_updates",
+			Help: "Total number of updates per configmap",
+		},
+		[]string{"name", "namespace", "cmkey", "status"},
```

Comment:
`cmkey` is a high-cardinality risk for large setups. It should be sufficient to output in the log instead.